.github/workflows/coverage.yml

```diff
@@ -17,7 +17,7 @@ env:
 
 jobs:
   coverage:
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-18.04
```

Comment:
hopefully no package name changes!

Reply:
it's why this PR exists: it's a try :p

Reply:
Docker is the way Nico! I quickly boot docker Ubuntu 18.04 image and see package name. You save some CI minutes 😎

Reply:
Not so eco-friendly heh

Reply:
@alkino  : `libfl-dev` needs to be installed on Ubuntu 18.04

Reply:
Yes, I already have been trapped inside nmodl for that